package.json

```diff
@@ -17,7 +17,6 @@
     "pretest:e2e": "npm run build && npm run tolgee:start",
     "posttest:e2e": "npm run tolgee:stop",
     "test:e2e": "cross-env NODE_OPTIONS=\"--experimental-vm-modules\" jest -c jest.e2e.config.ts --runInBand",
-    "test:e2e-run": "cross-env NODE_OPTIONS=\"--experimental-vm-modules\" jest -c jest.e2e.config.ts --runInBand",
```

Comment:
same as "test:e2e", so I remove it

Reply:
It is necessary. It's not the same because it does not have pre/post tasks, and CI uses it (so it sets up the docker container itself).

Reply:
Got, fixed